.github/ISSUE_TEMPLATE/feature_request_template.yml

```diff
@@ -1,14 +1,28 @@
-name: ✨ Feature
-description: Request a new feature
-title: "[Feat] <title>"
+name: ✨ Feature Request
```

Comment:
Should we include a section for "Acceptance Criteria"?

Reply:
I think so.  On all of them, please!